inflammation-analysis.py

```diff
@@ -27,7 +27,8 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
```

Comment:
Standard Deviation for one patient with multiple observations

Reply:
Great help!